### PR TITLE
Implement user permission system

### PR DESCRIPTION
**Pull request by Brutus5000**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ before_install:
 install:
   - git clone https://github.com/FAForever/faf-stack.git faf-stack
     && pushd faf-stack
-    && git checkout e33243c9
+    && git checkout 79c5d9d9
```
Comment (norraxx) [inline .travis.yml]: Maybe let's start using branch names? At least it will be visible what commits do we need.
It's propositon to make things better  :-)

Comment (Brutus5000) [reply .travis.yml]: Branch names aren't fixed. You could not rerun and older pipeline if the branch (e.g. develop) has changed over time.

Comment (louvenarde) [reply .travis.yml]: (Maybe with tags then?)

Comment (Brutus5000) [reply .travis.yml]: Well you can use tags already, but the problem is the development phase when there is no tag yet.

**Pull request merged**